frontend/awx/resources/templates/WorkflowJobTemplatePage/WorkflowJobTemplatePage.tsx

```diff
@@ -49,9 +47,24 @@ export function WorkflowJobTemplatePage() {
   });
   const error = isNotifAdminError || templateError;
 
+  const tabs: { label: string; page: string }[] = useMemo(() => {
+    const tabs = [
+      { label: t('Details'), page: AwxRoute.WorkflowJobTemplateDetails },
+      { label: t('Access'), page: AwxRoute.WorkflowJobTemplateAccess },
+      { label: t('Schedules'), page: AwxRoute.WorkflowJobTemplateSchedules },
+      { label: t('Jobs'), page: AwxRoute.WorkflowJobTemplateJobs },
+      { label: t('Survey'), page: AwxRoute.WorkflowJobTemplateSurvey },
+    ];
+    if (currentUser?.is_system_auditor || (isNotifAdmin && isNotifAdmin.results.length > 0)) {
```

Comment:
this can be inlined in the tabs definition above now, defaulting to false if the condition for the tab isn't met